relayer/relayer.go

```diff
@@ -37,30 +42,26 @@ func (r *Relayer) Start(stop <-chan struct{}, sysErr chan error) {
 	log.Debug().Msgf("Starting relayer")
 	messagesChannel := make(chan *Message)
 
-	// init new instance of ChainMetrics
-	chainMetrics := metrics.NewChainMetrics()
-
-	// init new mux router
-	router := mux.NewRouter()
-
-	// register path + handler
-	router.Path("/metrics").Handler(promhttp.Handler())
-
-	// start http server in non-blocking goroutine
-	go func() {
-		log.Fatal().Err(http.ListenAndServe(":2112", router))
-	}()
-	log.Debug().Msg("listening on: http://localhost:2112/metrics")
-
 	for _, c := range r.relayedChains {
 		log.Debug().Msgf("Starting chain %v", c.DomainID())
 		r.addRelayedChain(c)
 		go c.PollEvents(stop, sysErr, messagesChannel)
 	}
+
+	metrics, err := opentelemetry.InitializeMetrics(otlpmetrichttp.WithEndpoint("localhost:4318"), otlpmetrichttp.WithInsecure())
```

Comment:
@P1sar What is the planned way to allow for configuration from user in Start ?
Some config struct passed into Start()? Env?

Reply:
Just a reminder: As we discussed on planning, create the separate issue for Configuration refactoring so we can feet general relayer config params in to it.